src/argilla_llama_index/llama_index_handler.py

```diff
@@ -232,384 +233,261 @@ def _add_context_fields(self, number_of_retrievals: int) -> List[Any]:
     def _add_context_questions(self, number_of_retrievals: int) -> List[Any]:
         """Create the context questions to be added to the dataset."""
         rating_questions = [
-            rg.RatingQuestion(
+            RatingQuestion(
                 name=f"rating_retrieved_document_{doc + 1}",
-                title=f"Rate the relevance of the Retrieved document {doc + 1} (if present)",
+                title=f"Rate the relevance of the Retrieved document {doc + 1}, if applicable.",
                 values=list(range(1, 8)),
-                description=f"Rate the relevance of the retrieved document {doc + 1}.",
+                description=f"Rate the relevance of the retrieved document {doc + 1}, if applicable.",
                 required=False,
             )
             for doc in range(number_of_retrievals)
         ]
         return rating_questions
 
-    def _create_root_and_other_nodes(self, trace_map: Dict[str, List[str]]) -> None:
-        """Create the root node and the other nodes in the tree."""
-        self.root_node = self._get_event_name_by_id(trace_map["root"][0])
-        self.event_ids_traced = set(trace_map.keys()) - {"root"}
-        self.event_ids_traced.update(*trace_map.values())
-        for id in self.event_ids_traced:
-            self.components_to_log.add(self._get_event_name_by_id(id))
-
-    def _get_event_name_by_id(self, event_id: str) -> str:
-        """Get the name of the event by its id."""
-        return str(self.events_data[event_id][0].event_type).split(".")[1].lower()
-
-        # TODO: If we have a component more than once, properties currently don't account for those after the first one and get overwritten
+    def class_name(cls) -> str:
+        """Class name."""
+        return "ArgillaSpanHandler"
```

Comment:
I think we can use builtin methods for this. Something like `self.__class__.__name__`

Reply:
This was to maintain the convention as here: https://docs.llamaindex.ai/en/stable/api_reference/instrumentation/span_handlers/#llama_index.core.instrumentation.span_handlers.simple.SimpleSpanHandler